contrib/nitrpg/src/statistics.nit

```diff
@@ -78,6 +69,97 @@ redef class Player
 	end
 end
 
+redef class Counter[E]
+	redef fun inc(e) do
+		self[e] = self[e] + 1
```

Comment:
Because the Counter::[] is able to treat non existing keys and return 0 but not the `inc`implementation that calls directly `self.map[e]`.